### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 2nd March 15:00UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#974]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - MoneroKon 2024 Submission Deadline - 2 weeks™ - Submit a proposal for talk/workshop before 15th March - https://apply.monerokon.org/
    - https://kycnot.me/ UI revamped
    - Vtnerd funded! https://monerofund.org/projects/Q1Q2_2024_dev_vtnerd
    - Bootstrappable builds draft by tobtoht ready for tests https://github.com/monero-project/monero/pull/8929
    - Monero core/gui point release soon
    - https://www.reddit.com/r/Monero/comments/1b24066/integration_of_ethxmr_atomic_swaps_into_a_wallet/
    - hashrate 3 year low shared by sech1 in #monero-pow https://bitinfocharts.com/comparison/monero-hashrate.html#3y
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/) [j-berman update-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/426#note_23419)  / [v1docq47](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/420#note_23390) / [OxFFC](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/429#note_23450)
  a. [Boog900 full time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/431)    
  b. [jeffro256 full-time dev 2024Q2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/436)    
   c. [Unnamed Monero Wallet development](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/437)

6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - 
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/968)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Morpheus of Trocador wishes to share an update of sorts:

"""
It seemed to me an attack on the Monero project. Binance delisting, while disabling withdrawals and at the same time telling everyone that perhaps it would not be able to pay the price for the remaining Monero, had all the characteristics of an attack. By telling it would not be able to pay the price in USD for the remaining Monero, it was creating insecurity on investors, who would only have the alternative to sell it. To me, seems like Binance was forced into pressuring the crypto community to abandon Monero. If this was an attack, however, it was very stupid, because instead of having only one major centralized entity negociating Monero, now that market share was taken by multiple smaller swappers/exchanges, who are profiting more from it. So by attacking and delisting Monero, the entity behind this attack ceded his dominance over the market and decentralized the community even further, making it stronger, really. If someone wanted to attack Monero, they should have made easier to transact it on Binance, where they could not only manipulate price, but also control information.

What would people think if Monero went back to 160 USD? after all this drama, having no effect on price? demand would show up, because it would have been proven the antifragility of Monero. Then, if people want to buy it, where are they gonna get it from?

So for us, our partners are still able to find and sell monero, and to buy it back as well. In the end, even if all major CEXs delist Monero, all minor exchanges would still be able to find liquidity with private people, such as hodlers and whales. In the end, people that hold large sums of Monero would still be able to provide liquidity to ChangeNow, SimpleSwap, etc. So governments would have to ban Monero itself, a.k.a., banning free speech, in order to make this illegal. And even if they do that, they only prove the point of the need for Monero.

"""

**Comment by plowsof** [comment]
Majesticbank shared a comment about liquidity / volume post Byenance:
https://github.com/monero-project/monero-site/issues/2247#issuecomment-1962941440

"""
I can speak about the volume itself, which goes up to x3 from the same period of last year.

MajesticBank was never dependent on centralized exchanges liquidity in first place,

Hopefully with spot market and two more integrations with monero-only wallets we are closing the question of liquidity
"""

**Comment by plowsof** [comment]
Logs 
> __< p​lowsof:matrix.org >__ Meeting in 1 hr 20~ mins https://github.com/monero-project/meta/issues/974     

> __< t​rasherdk:monero.social >__ If you doing the how-to, and only one less request for how to do it, it's a win.     

> __< t​rasherdk:monero.social >__ When I gave a fuck about reddit, you'd have at least 5 asking about how to buy XMR anonymously.     

> __< t​rasherdk:monero.social >__ That's why I left.     

> __< t​rasherdk:monero.social >__ Oh, and the "cake solves this" was tha second reason.     

> __< s​hortwavesurfer2009:monero.social >__ https://monero.town/post/2316845     

> __< p​lowsof:matrix.org >__ Meeting time https://github.com/monero-project/meta/issues/974     

> __< p​lowsof:matrix.org >__ Greetings!     

> __< t​obtoht:monero.social >__ Hi     

> __< p​lowsof:matrix.org >__ whats been going on since the last meeting? hi tobtoht, how are things going with your bootstrappable builds PR?     

> __< p​lowsof:matrix.org >__ and i believe there is also a new feather wallet release coming soon, to fix a tor / crashing issue with latest version of tails?     

> __< p​lowsof:matrix.org >__ Bootstrappable builds draft by tobtoht ready for tests https://github.com/monero-project/monero/pull/8929     

> __< m​ichael:monero.social >__ Hello.     

> __< p​lowsof:matrix.org >__ detailed featherwallet updates/reports are always shared in #feather:monero.social , v 2.6.3 latest https://featherwallet.org/changelog/     

> __< p​lowsof:matrix.org >__ Vtnerd funded! https://monerofund.org/projects/Q1Q2_2024_dev_vtnerd , (Rucknium made the neccassary changes to get the MAGIC monerofund displayed inside feather by tobs , and received a large % of donations directly from FW users 🫡)     

> __< p​lowsof:matrix.org >__ MoneroKon 2024 Submission Deadline - 2 weeks™ Submit a proposal for talk/workshop before 15th March - https://apply.monerokon.org/ ajs_ MSvB     

> __< p​lowsof:matrix.org >__ a lowlight (but a highlight for the remaining miners) sech1 noted a 3 year low in monero hashrate https://bitinfocharts.com/comparison/monero-hashrate.html#3y     

> __< p​lowsof:matrix.org >__ https://bounties.monero.social/ -> #monero-bounties:monero.social has been piling up funds for many tasks lately     

> __< p​lowsof:matrix.org >__ pigeons is almost there with https://translate.getmonero.org/ (currently offline) 😄 but is awaiting settings to be ported over     

> __< p​lowsof:matrix.org >__ News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero Moon](https://www.themoneromoon.com/)     

> __< p​lowsof:matrix.org >__ and just to share a message/update from Trocador/Morpheus 'post Byenance' https://github.com/monero-project/meta/issues/974#issuecomment-1974799436     

> __< t​obtoht:monero.social >__ Would love to hear feedback from people that usually participate in the gitian.sigs process. Plowsof, Lyza, Github CI and myself were already to reproducibly build #8929. If you run a build, please share your hashes too. You can find instructions in README.md. Reviews on any of the sub-PRs are also greatly appreciated to help move this forward.     

> __< t​obtoht:monero.social >__ Builds for testing can be found here: https://github.com/monero-project/monero/actions/runs/8047961641     

> __< p​lowsof:matrix.org >__ hyc sech1 would be interested, they battle for who can build the gitian binaries first :D     

> __< p​lowsof:matrix.org >__ another post byenance update from Majesticbank https://github.com/monero-project/meta/issues/974#issuecomment-1974800144     

> __< p​lowsof:matrix.org >__ it would seem that business is booming for instant crypto swap sites     

> __< r​ottenwheel:kernal.eu >__ Does that annoy you too?     

> __< t​obtoht:monero.social >__ Fix was included in 2.6.3. This issue also affects GUI, so I've upstreamed the patch. It should make it into the upcoming release.     

> __< p​lowsof:matrix.org >__ speaking of cake , i noticed a tweet but twiiit is unable to bring it up     

> __< p​lowsof:matrix.org >__ selsta presented quite the merge list to luigi the other day. monero core/gui release incoming     

> __< selsta >__ yes, I wanted to see if we can fix the RandomX crash on Windows but seems that's a bit more complex     

> __< selsta >__ so we will release it without the fix     

> __< t​rasherdk:monero.social >__ You know, when you see it, like 10 times a day, it will get like, why do I look at the same question and answers every time I visit reddit. Yes, it annoys the shit out of me.     

> __< t​obtoht:monero.social >__ selsta: I'm out of ideas on that one     

> __< p​lowsof:matrix.org >__ thanks selsta, sech1 was investigating that issue closely with the he... ^ help of tobotht who is out of ideas now xD     

> __< r​ottenwheel:kernal.eu >__ Monero fixes this. 🎂     

> __< p​lowsof:matrix.org >__ any other things people wish to highlight?     

> __< p​lowsof:matrix.org >__ let me drop the recent and give a moment to anyone who wants to share something [CCS updates](https://ccs.getmonero.org/) [j-berman update-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/426#note_23419)  / [v1docq47](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/420#note_23390) / [OxFFC](https://repo.getmonero.org/monero-proje<clipped message>     

> __< p​lowsof:matrix.org >__ ct/ccs-proposals/-/merge_requests/429#note_23450)     

> __< p​lowsof:matrix.org >__ last but not least, vtnerd https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/435     

> __< p​lowsof:matrix.org >__ recent ccs updates*     

> __< p​lowsof:matrix.org >__ ok lets jump in to the CCS ideas r4v3r23 here?     

> __< p​lowsof:matrix.org >__ a. [Boog900 full time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/431)     

> __< p​lowsof:matrix.org >__ 5 solid updoots. the cuprate coordinator / architect / you name it. last update was a change in rates after requested last meeting     

> __< t​rasherdk:monero.social >__ rottenwheel: Too bad, monero can't fix reddit.     

> __< t​rasherdk:monero.social >__ Damn, that dude is everywhere.     

> __< r​ottenwheel:kernal.eu >__ TrasherDK reddit is a lost cause. Delete or abandon your account and join Lemmy via monero.town.     

> __< p​lowsof:matrix.org >__ thanks monerobull for maintaining that^     

> __< p​lowsof:matrix.org >__ anyone have a comment for cuprate boog900? or want to share their vote?     

> __< t​rasherdk:monero.social >__ jordan. Type faster.     

> __< d​iego:cypherstack.com >__ Is farcaster still a thing?     

> __< j​ordan_sanchez:matrix.org >__ I mean we can sort of fix Reddit :/     

> __< j​ordan_sanchez:matrix.org >__ They’re about to ipo, so we can buy up a majority of shares, & nominate our people onto the board of directors.     

> __< d​iego:cypherstack.com >__ Oh shoot its a meeting. Sorry.     

> __< p​lowsof:matrix.org >__ the ccs is "completed" Diego Salazar , im not yet aware of it in production anywhere but iirc ... basicswapdex have used it somehow to .. help them.. with.. a thing     

> __< t​rasherdk:monero.social >__ rottenwheel: I'm on the town already.     

> __< t​rasherdk:monero.social >__ It's seems like it's mostly spam.     

> __< t​rasherdk:monero.social >__ Like reddit.     

> __< p​lowsof:matrix.org >__ the ccs closed out with a MVP front end web UI on test/stagenet     

> __< r​ottenwheel:kernal.eu >__ TrasherDK will you ever like anything? You're starting to sound like someone we both know... 😉     

> __< p​lowsof:matrix.org >__ moving on then boog900, thank you for attending/reading 👍️     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)

**Comment by plowsof** [comment]
Logs 
> __< p​lowsof:matrix.org >__ Meeting in 1 hr 20~ mins https://github.com/monero-project/meta/issues/974     

> __< t​rasherdk:monero.social >__ If you doing the how-to, and only one less request for how to do it, it's a win.     

> __< t​rasherdk:monero.social >__ When I gave a fuck about reddit, you'd have at least 5 asking about how to buy XMR anonymously.     

> __< t​rasherdk:monero.social >__ That's why I left.     

> __< t​rasherdk:monero.social >__ Oh, and the "cake solves this" was tha second reason.     

> __< s​hortwavesurfer2009:monero.social >__ https://monero.town/post/2316845     

> __< p​lowsof:matrix.org >__ Meeting time https://github.com/monero-project/meta/issues/974     

> __< p​lowsof:matrix.org >__ Greetings!     

> __< t​obtoht:monero.social >__ Hi     

> __< p​lowsof:matrix.org >__ whats been going on since the last meeting? hi tobtoht, how are things going with your bootstrappable builds PR?     

> __< p​lowsof:matrix.org >__ and i believe there is also a new feather wallet release coming soon, to fix a tor / crashing issue with latest version of tails?     

> __< p​lowsof:matrix.org >__ Bootstrappable builds draft by tobtoht ready for tests https://github.com/monero-project/monero/pull/8929     

> __< m​ichael:monero.social >__ Hello.     

> __< p​lowsof:matrix.org >__ detailed featherwallet updates/reports are always shared in #feather:monero.social , v 2.6.3 latest https://featherwallet.org/changelog/     

> __< p​lowsof:matrix.org >__ Vtnerd funded! https://monerofund.org/projects/Q1Q2_2024_dev_vtnerd , (Rucknium made the neccassary changes to get the MAGIC monerofund displayed inside feather by tobs , and received a large % of donations directly from FW users 🫡)     

> __< p​lowsof:matrix.org >__ MoneroKon 2024 Submission Deadline - 2 weeks™ Submit a proposal for talk/workshop before 15th March - https://apply.monerokon.org/ ajs_ MSvB     

> __< p​lowsof:matrix.org >__ a lowlight (but a highlight for the remaining miners) sech1 noted a 3 year low in monero hashrate https://bitinfocharts.com/comparison/monero-hashrate.html#3y     

> __< p​lowsof:matrix.org >__ https://bounties.monero.social/ -> #monero-bounties:monero.social has been piling up funds for many tasks lately     

> __< p​lowsof:matrix.org >__ pigeons is almost there with https://translate.getmonero.org/ (currently offline) 😄 but is awaiting settings to be ported over     

> __< p​lowsof:matrix.org >__ News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero Moon](https://www.themoneromoon.com/)     

> __< p​lowsof:matrix.org >__ and just to share a message/update from Trocador/Morpheus 'post Byenance' https://github.com/monero-project/meta/issues/974#issuecomment-1974799436     

> __< t​obtoht:monero.social >__ Would love to hear feedback from people that usually participate in the gitian.sigs process. Plowsof, Lyza, Github CI and myself were already to reproducibly build #8929. If you run a build, please share your hashes too. You can find instructions in README.md. Reviews on any of the sub-PRs are also greatly appreciated to help move this forward.     

> __< t​obtoht:monero.social >__ Builds for testing can be found here: https://github.com/monero-project/monero/actions/runs/8047961641     

> __< p​lowsof:matrix.org >__ hyc sech1 would be interested, they battle for who can build the gitian binaries first :D     

> __< p​lowsof:matrix.org >__ another post byenance update from Majesticbank https://github.com/monero-project/meta/issues/974#issuecomment-1974800144     

> __< p​lowsof:matrix.org >__ it would seem that business is booming for instant crypto swap sites     

> __< r​ottenwheel:kernal.eu >__ Does that annoy you too?     

> __< t​obtoht:monero.social >__ Fix was included in 2.6.3. This issue also affects GUI, so I've upstreamed the patch. It should make it into the upcoming release.     

> __< p​lowsof:matrix.org >__ speaking of cake , i noticed a tweet but twiiit is unable to bring it up     

> __< p​lowsof:matrix.org >__ selsta presented quite the merge list to luigi the other day. monero core/gui release incoming     

> __< selsta >__ yes, I wanted to see if we can fix the RandomX crash on Windows but seems that's a bit more complex     

> __< selsta >__ so we will release it without the fix     

> __< t​rasherdk:monero.social >__ You know, when you see it, like 10 times a day, it will get like, why do I look at the same question and answers every time I visit reddit. Yes, it annoys the shit out of me.     

> __< t​obtoht:monero.social >__ selsta: I'm out of ideas on that one     

> __< p​lowsof:matrix.org >__ thanks selsta, sech1 was investigating that issue closely with the he... ^ help of tobotht who is out of ideas now xD     

> __< r​ottenwheel:kernal.eu >__ Monero fixes this. 🎂     

> __< p​lowsof:matrix.org >__ any other things people wish to highlight?     

> __< p​lowsof:matrix.org >__ let me drop the recent and give a moment to anyone who wants to share something [CCS updates](https://ccs.getmonero.org/) [j-berman update-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/426#note_23419)  / [v1docq47](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/420#note_23390) / [OxFFC](https://repo.getmonero.org/monero-proje<clipped message>     

> __< p​lowsof:matrix.org >__ ct/ccs-proposals/-/merge_requests/429#note_23450)     

> __< p​lowsof:matrix.org >__ last but not least, vtnerd https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/435     

> __< p​lowsof:matrix.org >__ recent ccs updates*     

> __< p​lowsof:matrix.org >__ ok lets jump in to the CCS ideas r4v3r23 here?     

> __< p​lowsof:matrix.org >__ a. [Boog900 full time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/431)     

> __< p​lowsof:matrix.org >__ 5 solid updoots. the cuprate coordinator / architect / you name it. last update was a change in rates after requested last meeting     

> __< t​rasherdk:monero.social >__ rottenwheel: Too bad, monero can't fix reddit.     

> __< t​rasherdk:monero.social >__ Damn, that dude is everywhere.     

> __< r​ottenwheel:kernal.eu >__ TrasherDK reddit is a lost cause. Delete or abandon your account and join Lemmy via monero.town.     

> __< p​lowsof:matrix.org >__ thanks monerobull for maintaining that^     

> __< p​lowsof:matrix.org >__ anyone have a comment for cuprate boog900? or want to share their vote?     

> __< t​rasherdk:monero.social >__ jordan. Type faster.     

> __< d​iego:cypherstack.com >__ Is farcaster still a thing?     

> __< j​ordan_sanchez:matrix.org >__ I mean we can sort of fix Reddit :/     

> __< j​ordan_sanchez:matrix.org >__ They’re about to ipo, so we can buy up a majority of shares, & nominate our people onto the board of directors.     

> __< d​iego:cypherstack.com >__ Oh shoot its a meeting. Sorry.     

> __< p​lowsof:matrix.org >__ the ccs is "completed" Diego Salazar , im not yet aware of it in production anywhere but iirc ... basicswapdex have used it somehow to .. help them.. with.. a thing     

> __< t​rasherdk:monero.social >__ rottenwheel: I'm on the town already.     

> __< t​rasherdk:monero.social >__ It's seems like it's mostly spam.     

> __< t​rasherdk:monero.social >__ Like reddit.     

> __< p​lowsof:matrix.org >__ the ccs closed out with a MVP front end web UI on test/stagenet     

> __< r​ottenwheel:kernal.eu >__ TrasherDK will you ever like anything? You're starting to sound like someone we both know... 😉     

> __< p​lowsof:matrix.org >__ moving on then boog900, thank you for attending/reading 👍️     

> __< t​rasherdk:monero.social >__ Oh, shit. Did we just spam a community meeting. I'm sorry 😂     

> __< r​4v3r23:monero.social >__ sup     

> __< p​lowsof:matrix.org >__ hi r4v3r23, after jeffros proposal, it would be nice if you could share thoughts, if any, on the last one     

> __< p​lowsof:matrix.org >__ b. [jeffro256 full-time dev 2024Q2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/436)     

> __< p​lowsof:matrix.org >__ when is jeffro256 going to end his current ccs... been a busy bee lately     

> __< t​obtoht:monero.social >__ obvious merge imo     

> __< d​iego:cypherstack.com >__ I have an update and a request regarding Cypher Stack's active CCS on BP++ for whenever it's time for that. :)     

> __< p​lowsof:matrix.org >__ Rucknium asked about more serpahis work and this ccs claims to be doing that     

> __< t​rasherdk:monero.social >__ Dude, I like a lot of things.     

> __< t​rasherdk:monero.social >__ I'm hanging out in the "no wallet" and "MRL.." channels, and even though they are breaking my brain, I still read every single post.     

> __< t​rasherdk:monero.social >__ And yes, I'm an alt for that guy we both know, for sure 😂     

> __< t​rasherdk:monero.social >__ I'm not hiding, and it's not that hard to find my real world name.     

> __< p​lowsof:matrix.org >__ thanks Diego, awesome, one new proposal to share but as you  can see the naughty people are just talking through us anyway     

> __< r​ottenwheel:kernal.eu >__ Quit spamming the meeting! 🚨     

> __< p​lowsof:matrix.org >__ "obvious merge" sounds like some high praise. we can move on, its only been  up for 4 days.. just waiting for jeff to end out the prev ccs i guess     

> __< p​lowsof:matrix.org >__ c. [Unnamed Monero Wallet development](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/437)     

> __< t​rasherdk:monero.social >__ rottenwheel: Yeah, you too 😂     

> __< p​lowsof:matrix.org >__ r4v3r23: could perhaps share some feedback now.. or later (it was posted 18@hrs ago and there is some history here) apparently this is the former Anonero wallet developer but they have now parted ways. Unnamed Monero Wallet is the ... "dev has gone solo" name of his first album     

> __< r​4v3r23:monero.social >__ heres the deal: cyjan was hired a few months ago to work on monero-side code/features to anonero, and we agreed to build a whole new architecture from the ground up for a v1.0 rewrite     

> __< r​4v3r23:monero.social >__ hes not "the" dev - he was brought on later to help out     

> __< r​4v3r23:monero.social >__ due to to some disagreements, so they took anonero v1.0, changed the color theme, and rebranded     

> __< r​4v3r23:monero.social >__ the current xmruw wallet is just a copy (feature wise) of anonero (including planned features)     

> __< r​4v3r23:monero.social >__ i dont see why the wallet should be funded since they just copied us, but the underlying monero_c lib is interesting     

> __< p​lowsof:matrix.org >__ understood , thank you for the feedback. i think its too early to explore / comment on the goals of this ccs now as it needs more eyes.     

> __< p​lowsof:matrix.org >__ Diego Salazar: would you like to share an update on CypherStacks BP++ peer review?     

> __< r​4v3r23:monero.social >__ yeah, i think its worth separating the lib from the wallet itself     

> __< p​lowsof:matrix.org >__ and then the ecosystem can more easily 'use' it right?     

> __< p​lowsof:matrix.org >__ ill make sure to pass your comments on and wait for a response     

> __< p​lowsof:matrix.org >__ an update AND a request Diego Salazar 😮     

> __< r​4v3r23:monero.social >__ as it stands now, cyjan is lumping them together. xmruw wallet is written in flutter/dart - thats project specific and not useful for the ecosysem     

> __< r​4v3r23:monero.social >__ but the monero_c lib that its based on can be used by anyone     

> __< r​4v3r23:monero.social >__ as it stands now, cyjan is lumping them together. xmruw wallet is written in flutter/dart - thats project specific and not useful for other projects to use     

> __< t​rasherdk:monero.social >__ It should be an easy choice. r4v3r23: is a well known entity, while the fork dude is totally unknown to me, at least.     

> __< t​rasherdk:monero.social >__ unknown, to me at least.     

> __< d​iego:cypherstack.com >__ Hi     

> __< m​ichael:monero.social >__ Hi.     

> __< d​iego:cypherstack.com >__ We're well over halfway done with the review. Expect it to be done this month.     

> __< d​iego:cypherstack.com >__ There have been several questions which we have contacted the authors about. They responded to a few of them and we've even fixed some incorrect proofs.     

> __< d​iego:cypherstack.com >__ So being good science citizens along the way beyond just a review.     

> __< d​iego:cypherstack.com >__ This type of review typically has milestones but given the proposal itself wasn't swt up by us, but rather we were contracted to do it, we didn't set up the terms. I should have looked at them more carefully admittedly.     

> __< p​lowsof:matrix.org >__ https://repo.getmonero.org/monero-project/ccs-proposals/-/blob/master/bulletproofs-pp-peer-review.md for reference     

> __< d​iego:cypherstack.com >__ Which brings me to the request. I'd like to request 50% of the total be paid out to us since we're over halfway done. Since this was such a long project, that money would go a long way in liquid cash for payroll and stuff.     

> __< r​4v3r23:monero.social >__ ive worked with cyjan and can vouch for them. technical people can give their opinions on the usefulness of the monero_c lib. the issue I have with them is work related, not code related.      

> __< r​4v3r23:monero.social >__ also, the asking rate of $125/XMR is a piss take     

> __< d​iego:cypherstack.com >__ Having 32k "on hold" until completion on a several month project can be a bit painful.     

> __< d​iego:cypherstack.com >__ Just a request. If the community thinks no then we'll make it work somehow.     

> __< p​lowsof:matrix.org >__ not to mention, originally, the contract was for another paper, which was worked on, then a new paper was released (larger in size) causing more delays?     

> __< d​iego:cypherstack.com >__ Right. The new paper more than doubled the time needed for review.     

> __< p​lowsof:matrix.org >__ so would anyone like to vote on providing 50% of the funds upfront? because i was 'there' for the delays since the inception of this proposal, i would say of course     

> __< s​unchakr:monero.social >__ Hey all! We are live now!  to talk about Finny Forum w/ RagnarLif  + Price Report w/ BawdyAnarchist_, News w/ tony_huszar & MORE!     

> __< s​unchakr:monero.social >__ 👀➡️: youtube.com/watch?v=ZkMuKX…     

> __< s​unchakr:monero.social >__ Join ➡️: streamyard.com/5zwbg7ptmm     

> __< s​unchakr:monero.social >__ Join us if you can 😊     

> __< p​lowsof:matrix.org >__ jberman could speak more to the validity of the request if possible     

> __< r​ub1ck:matrix.org >__ yt link not working     

> __< p​lowsof:matrix.org >__ do you want to stop Dr Aaron / Sarang from starving yes/no     

> __< p​lowsof:matrix.org >__ xD     

> __< s​unchakr:monero.social >__ https://www.youtube.com/live/ZkMuKXIbo1Q?si=OPesXBWa_wggpcxm     

> __< r​ub1ck:matrix.org >__ thanks, this is working. watching it     

> __< s​unchakr:monero.social >__ Awesome 😊     

> __< t​rasherdk:monero.social >__ r4v3r23: I did not take the time to look a their CCS request. I'm pretty sure, if their direction is the best path, you'd go that way.     

> __< t​rasherdk:monero.social >__ I'm not in favor of supporting 2 versions of the same thing, just because you have differences. That was the soft version 😇     

> __< t​obtoht:monero.social >__ I'm not sure if building on `wallet2_api.h` at this stage in seraphis development is worthwhile. We don't know how much of it will have to be added, binned or rewritten - or if this interface will even exist post-Seraphis. https://github.com/seraphis-migration/wallet3/issues/5     

> __< p​lowsof:matrix.org >__ thanks for the update/request Diego Salazar feedback shall arrive in due time. i could or if you like, comment to the same effect under the ccs proposal     

> __< p​lowsof:matrix.org >__ thank you all for attending !     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)


continued:
```
16:27:13 <mrcyjanek> Was my CCS proposal (unnamed monero wallet) already discussed or not? I didnt find the irc logs yet
16:30:28 <m-relay> <a​js_:matrix.org> Why is the wallet “unnamed” out of curiosity
16:32:36 <mrcyjanek_> Naming things is hard and 'unnamed' felt just okay, somebody suggested me nameless over in chatroom on telegram but that would be too close to anonero.
16:42:49 <m-relay> <t​rasherdk:monero.social> I understand, it's hart to find a name, for a fork, of an already running project, without being too, what's the word I'm looking for, [I can't find a nice word in my vocabulary]..
16:52:11 <mrcyjanek__> I've seen that r​4v3r23 already mentioned few details. I knoe that I am not on time but I didnt know my CCS proposal will be discussed here :/. I'll get matrix account as soon as I get better internet access
16:52:36 <mrcyjanek__> I'm open to all kind of questions here / privately.
16:58:01 <m-relay> <t​rasherdk:monero.social> Well, this is where CSS proposals get discussed, so if you have anything to say, in addition, this is the place.
17:16:45 <m-relay> <p​lowsof:matrix.org> sorry for not commenting on your proposal about the meeting mrcyjanek
```
